djweed/db_fields.py

```diff
@@ -1,5 +1,8 @@
 from django.contrib.contenttypes.models import ContentType
-from django.core.urlresolvers import reverse
+try:
+    from django.core.urlresolvers import reverse
+except ImportError:  # Django 1.8
```

Comment:
I've checked with Django 1.11.1 where I actually use django-weed and there is `reverse` in `django.core.urlresolvers`. So according to notification on this page https://docs.djangoproject.com/en/1.11/ref/urlresolvers/ this change is for Django 2.0 compatibility, not for 1.8, until 2.0 it is working (I guess I want the comment to be more correct)